pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -221,9 +221,10 @@ func (a *ApplicationBackupController) updateWithAllNamespaces(backup *stork_api.
 	if err != nil {
 		return fmt.Errorf("error updating with all namespaces for wildcard: %v", err)
 	}
+	pxNs, _ := utils.GetPortworxNamespace()
```

Comment:
We already evaluated the pxNs in the handle method , can we pass the pxNs to this method or persist in ApplicationBackupController?. So no need for reevaluating the pxNs , cause listing all services in a cluster is an expensive operation

Reply:
Both label and all namespace can not be set together?

Reply:
No both cannot be set together

Reply:
> We already evaluated the pxNs in the handle method , can we pass the pxNs to this method or persist in ApplicationBackupController?. So no need for reevaluating the pxNs , cause listing all services in a cluster is an expensive operation

Yes this would have been the ideal solution but there are possibilities that a customer a person is having stork but not portworx initially and later he might have it. In that scenario is it wise to store it?

Reply:
> Both label and all namespace can not be set together?

No both cannot be set together.

Reply:
> @vikasit12 I think, @px-kesavan is referring to avoid the call, for each reconcile cycle. In the case namespace label, we are calling functions to fetch the namespace everytime in the handle. That means it will be called every reconciler cyle. You can avoid it by checking if namespace label is present and namespace list is empty, then fetch the namespace.

Makes sense and changes have been made accordingly